Apache-Beam-Jobs.md

```diff
@@ -670,6 +671,28 @@ The code above throws this error `'_UnwindowedValues' object is not subscriptabl
     ))
 ```
 
+### `_namedptransform is not iterable` error
+
+This error sometimes happens when you forget to add a label for some operation
+(the strings of code before `>>`).  The solution is to add a label for all operations. 
```

Comment:
In the wiki we keep whole paragraphs on one line so that diffs don't show a bunch of changes just becase the text was re-wrapped

Reply:
done